lib/app/Auth/PhoenixTransactionBridge.php

```diff
@@ -22,6 +22,12 @@ public function getHeader($name) {
    * @return void
    */
   public function log($message, array $details) {
-    watchdog('PhoenixTransactionBridge', $message, $details, WATCHDOG_INFO);
+    $details = implode(', ', array_map(
+        function ($value, $key) { return sprintf("%s='%s'", $key, $value); },
+        $details,
+        array_keys($details)
```

Comment:
@angaither haha yeah it was something that I saw on Stackoverflow - basically what it does it turn the entire array (both the keys and values) into a string so that it can be added in the message mimicking how it does in Laravel logs. I can try to write a more simple way to do this though!